libs/core/src/lib/card/card-header.component.ts

```diff
@@ -21,11 +23,15 @@ import { CardSubtitleDirective } from './card-subtitle.directive';
     templateUrl: './card-header.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CardHeaderComponent implements OnInit, CssClassBuilder, AfterContentInit {
+export class CardHeaderComponent implements OnInit, OnChanges, CssClassBuilder, AfterContentInit {
+    /** Whether card header is interactive */
+    @Input()
+    interactive = true;
+
     /** @hidden */
     @Input()
     @HostBinding('attr.tabindex')
-    tabindex = '0';
+    tabindex = this.interactive ? '0' : '-1';
```

Comment:
If `interactive="false"` should take precedence over `tabindex` input it might be better to have `tabindex` as `getter setter`.
```
@Input()
@HostBinding('attr.tabindex')	
set tabindex(tabindex: string) {
  this._tabindex =  tabindex;
}
get tabindex(): string {
  return !this.interactive ? '-1' : this._tabindex;
}
private _tabindex = '0';
```
Otherwise the user is able to override `tabindex` when `interactive="false"` just assigning tabindex  later.

If `tabindex` must be always based on `interactive` value then we shouldn't have tabindex as `input`;

@JKMarkowski  what do you think?

Reply:
Hi @dimamarksman you are totally right there, I will add setters/getters, just to let user decide whether this should be tabbable

Reply:
In this case we don't need this assinment in ngOnChanges:
```
if ('interactive' in changes) {
   this.tabindex = this.interactive ? '0' : '-1' ;
}
```